src/components/views/elements/DeviceVerifyButtons.js

```diff
@@ -65,7 +65,8 @@ export default React.createClass({
                         <ul>
                             <li><label>Device name:</label> <span>{ this.state.device.getDisplayName() }</span></li>
                             <li><label>Device ID:</label>   <span><code>{ this.state.device.deviceId}</code></span></li>
-                            <li><label>Device key:</label>  <span><code><b>{ this.state.device.getFingerprint() }</b></code></span></li>
+                            <li><label>Device fingerprint:</label>  <span><code><b>{ this.state.device.getFingerprint() }</b></code></span></li>
```

Comment:
I pulled this because the refactoring of the verify device dialog made it a difficult merge anyway.  I would like to have a longer discussion at some point about the "streaming key" at some point!  As I pointed out in our chat a few weeks ago, I'm not terribly clear on why each user+device pairing doesn't have a unique streaming key (currently both keys are basically permanent).